policyengine_us/variables/gov/states/ky/tax/income/credits/ky_tuition_tax_credit.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class ky_tuition_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Kentucky tuition tax credits"  # Form 8863-K
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.KY
+
+    def formula(tax_unit, period, parameters):
+        tentative_tax_credit = add(
```

Comment:
Married filing separately filers are ineligible. The tax form doesn't include head of household as an eligible status; is that because Kentucky doesn't have it? The legal code only excludes mfs.

Reply:
Kentucky does not have hoh or widowed filing statuses: https://answerconnect.cch.com/document/jky0109013e2c83ade1d7/state/explanations/kentucky/filing-status

Reply:
How do we handle that elsewhere, e.g. in main rates? If we have a `ky_filing_status` variable let's use that in the eligibility.

Reply:
we have not had to use the filing status variable in KY yet (flat tax), will add `ky_filing_status` here